std/path.d

```diff
@@ -3902,6 +3902,48 @@ if (isConvertibleToString!Range)
     assert(testAliasedString!isValidPath("/foo/bar"));
 }
 
+/** Checks whether `path1` is a subpath of `path2`
+  *
+  * Params:
+  *     path1 = string or Range of characters
+  *     path2 = string or Range of characters
+  *
+  * Returns:
+  *     true if `path` is a subpath of `path2`
+  *     false otherwise
+*/
+bool isSubPath(Range)(Range path1, Range path2)
```

Comment:
Please use more descriptive parameter names than `path1` and `path2`.

Please also consider UFCS. In `path1.isSubPath(path2)`, is it immediately obvious which argument is the prefix and which is the longer path? Is a better function name and argument order possible?